src/canisters/frontend/ic-asset/src/sync.rs

```diff
@@ -44,7 +40,6 @@ pub async fn sync(
 
     let operations = assemble_synchronization_operations(project_assets, container_assets);
 
-    println!("Committing batch.");
```

Comment:
We've run into issues in the past with some of these calls taking a long time, or timing out.  These output statements might seem extraneous, but in that case they give us useful information.  Also, they give context to per-file output like this:
```
  /favicon.ico (15406 bytes) sha 4e8d31b50ffb59695389d94e393d299c5693405a12f6ccd08c31bcf9b58db2d4 is already installed
  /main.css (537 bytes) sha 75ac0c5aea719bb2b887fffbde61867be5c3a9eceab3d75619763c28735891cb is already installed
  /main.css (gzip) (299 bytes) sha b4879e7ba34e68b2965d626e48d772ce615e4f6b78b69cc8f2f91127ed18b850 is already installed
```

Reply:
Ah, overzealous removal of debug prints...